.gitignore

```diff
@@ -309,3 +309,5 @@ Resources/MapImages
 ### Nyanotrasen Specific
 Tools/gen_build_info.py
 
+buildclient.cmd
+buildserver.cmd
```

Comment:
why does this PR change the .gitignore?

Reply:
i don't know to be honest, is it bad?

Reply:
just do like git checkout origin/master .gitignore

Reply:
how?

Reply:
`git checkout origin/master .gitignore` will take the .gitignore file from your original master branch. Just run that command in any git-capable shell. Or alternatively, copy `.gitignore` from Nyano, if you have to resort to it.